core/sds-aspect-meta-model-java/src/main/java/io/openmanufacturing/sds/metamodel/loader/DefaultPropertyWrapper.java

```diff
@@ -17,13 +17,14 @@
 import java.util.Optional;
 
 import io.openmanufacturing.sds.metamodel.Characteristic;
+import io.openmanufacturing.sds.metamodel.Value;
 import io.openmanufacturing.sds.metamodel.impl.DefaultCharacteristic;
 import io.openmanufacturing.sds.metamodel.impl.DefaultProperty;
 
 public class DefaultPropertyWrapper extends DefaultProperty {
 
    private Characteristic characteristic;
-   private Optional<Object> exampleValue;
+   private Optional<Value> exampleValue;
```

Comment:
This is correct. Although it would be nice to remove this limitation, discussing/implementing this is out of scope here; I'll use `ScalarValue` for now.